bundles/org.openhab.binding.qbus/README.md

```diff
@@ -0,0 +1,114 @@
+# Qbus Binding
+
+![Qbus Logo](doc/Logo.JPG)
+
+This binding for [Qbus](https://qbus.be) communicates with all controllers of the Qbus home automation system.
+
+We also host a site which contains a [manual](https://manualoh.schockaert.tk/) where you can find lot's of information to set up openHAB with Qbus client and server (for the moment only in Dutch).
+
+The controllers can not communicate directly with openHAB, therefore we developed a client/server application which you must install prior to enable this binding.
+More information can be found here:
+[Qbus Client/Server]( https://github.com/QbusKoen/QbusClientServer-Installer)
+
+With this binding you can control and read almost every output from the Qbus system.
+
+## Supported Things
+
+The following things are supported by the Qbus binding:
+
+- `dimmer`: Dimmer 1 button, 2 button and clc
+- `onOff`: Bistabiel, Timer1-3, Interval
+- `thermostats`: Thermostats - normal and PID
+- `scene`: Scenes
+- `co2`: CO2 
+- `rollershutter`: Rollershutter 
+- `rollershutter_slats`: Rollerhutter with slats
+
+For now the following Qbus things are not yet supported but will come:
+
+- DMX
+- Timer 4 & 5
+- HVAC
+- Humidity
+- Renson
+- Duco
+- Kinetura
+- Energy monitor
+- Weather station
+
+
+## Discovery
+
+The discovery service is not yet implemented but the System Manager III software of Qbus generates things and item files from the programming, which you can use directly in openHAB.
+
+## Thing Configuration
+
+This is an example of the things configuration file:
+
+```
+Bridge qbus:bridge:CTD001122 [ addr="localhost", sn="001122", port=8447, refresh=10 ] {
+    dimmer                   1     "ToonzaalLED"      [ dimmerId=100 ]
+    onOff                    30    "Toonzaal230V"     [ bistabielId=76 ]
+    thermostat               50    "Service"          [ thermostatId=99 ]
+    scene                    70    "Disco"            [ sceneId=36 ]
+    co2                      100   "Productie"        [ co2Id=26 ]
+    rollershutter            120   "Roller1"          [ rolId=268 ]
+    rollershutter_slats      121   "Roller2"          [ rolId=264 ]
+}
+```
+
+
+| Property   | Default  | Required | Description    |
+|------------|----------|----------|----------------|
+| `addr`    | localhost | YES      | The ip address of the machine where the Qbus Server runs |
+| `sn`      |           | YES      | The serial number of your controller |
+| `port`    | 8447      | YES      | The communication port of the client/server |
+| `refresh` | 5         | NO       | Refresh time - After x minutes there will be a check if server is still running and if client is still connected. If not - reconnect |
```

Comment:
Can you fix the formatting of the tables? There are dome markdown table beautifiers on the net.